src/settings/StaffSlips/components/ViewSections/StaffSlipAboutSection/StaffSlipAboutSection.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  within,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import StaffSlipAboutSection from './StaffSlipAboutSection';
+
+describe('StaffSlipAboutSection', () => {
+  const testStaffSlip = {
+    name: 'testNameValue',
+    description: 'testDescriptionValue',
+  };
+
+  const getItemByTestId = (id) => within(screen.getByTestId(id));
+
+  const labelIds = {
```

Comment:
Could you please create const `testIds = {};` and add `data-testid` to this object?